### ISSUE TITLE
Illegal instruction: 4 swiftlint, sourcekitdInProc failed

### ISSUE THREAD

**Issue by DmitryKhristenko** [realm/SwiftLint#6249]
This error is related to a [closed issue](https://github.com/realm/SwiftLint/issues/5325 ), but [the solution](https://github.com/realm/SwiftLint/issues/1466#issuecomment-297265863 ) doesn't help.

### Bug Description
Build fails on stage _PhaseScriptExecution_. This error appeared after I upgraded my system to MacOS 26.0 and install Xcode 26 (release, 17A324).

**ERROR**
Command PhaseScriptExecution failed with a nonzero exit code.
SourceKittenFramework/library_wrapper.swift:58: Fatal error: Loading sourcekitdInProc.framework/Versions/A/sourcekitdInProc failed

...Library/Developer/Xcode/DerivedData/|||/Build/Intermediates.noindex/|||build/Debug-iphoneos/***.build/Script-F77239BF4C96CE14C7178444.sh: line 6: 15270 **Illegal instruction: 4  swiftlint**

### Environment

* SwiftLint version: 0.61.0
* Xcode version: 26.0, build version: 17A324
* Installation method used: Homebrew
* MacOS: 26.0 (25A354)
* Configuration file:

```yml
excluded:
  - ArkanaKeys/
  - ArkanaKeysInterfaces/
  - build/
  - Tuist/.build/

line_length: 150

```

Are you using [nested configurations](https://github.com/realm/SwiftLint#nested-configurations)?
- No.

script: """
                    if which swiftlint >/dev/null; then
                      swiftlint --fix && swiftlint
                    else
                      echo "Warning: SwiftLint not installed, download from https://github.com/realm/SwiftLint"
                    fi
                    """,
                    name: "Run SwiftLint",
                    inputFileListPaths: [
                        "build/build_phases/SwiftlintBuildPhase_swiftlint.xcfilelist",
                        "build/build_phases/SwiftlintBuildPhaseTests_swiftlint.xcfilelist",
                        "build/build_phases/SwiftlintBuildPhaseUITests_swiftlint.xcfilelist"
                    ],
                    outputPaths: [
                        "build/build_phases/SwiftlintBuildPhase_swiftlint_static_output",
                        "build/build_phases/SwiftlintBuildPhaseTests_swiftlint_static_output",
                        "build/build_phases/SwiftlintBuildPhaseUITests_swiftlint_static_output"
                    ],
                    basedOnDependencyAnalysis: true

**Comment by SimplyDanny** [comment]
Did you try [this](https://github.com/realm/SwiftLint/issues/1466#issuecomment-297265863)?

**Comment by debadaracco** [comment]
@DmitryKhristenko did you download the xcode from web? i had a similar issue with the web version and I fixed it with the app store version of the IDE.

**Comment by DmitryKhristenko** [comment]
> Did you try [this](https://github.com/realm/SwiftLint/issues/1466#issuecomment-297265863)?

Yes, I did. I wrote about trying this solution in my report.

**Comment by DmitryKhristenko** [comment]
> Did you try [this](https://github.com/realm/SwiftLint/issues/1466#issuecomment-297265863)?

I don't remember exactly, but what's the difference if the version is the same?

**Comment by debadaracco** [comment]
> > Did you try [this](https://github.com/realm/SwiftLint/issues/1466#issuecomment-297265863)?
> 
> I don't remember exactly, but what's the difference if the version is the same?

I don’t known but i tried after read this comment: https://github.com/realm/SwiftLint/issues/6221#issuecomment-3299086495

**Comment by SimplyDanny** [comment]
I doubt, the web version is any different from the App Store one. I tried with both and for me it works. If everything is configured properly, SwiftLint will be able to locate and load the dependency. In any case, there is nothing we can do about it.
